docs/development/local_development.md

```diff
@@ -0,0 +1,21 @@
+# Running on development mode
+
+The following document highlights how to deploy a Build operator locally for running on development mode.
+
+**Before generating an instance of the Build operator, ensure the following:**
+
+- Target your Kubernetes cluster. We recommend the usage of Kind for dev mode, you can see an our installation script in [install-kind.sh](/hack/install-kind.sh)
+- On the cluster, ensure Tekton controllers are running. You can use our Tekton installation script in [install-tekton.sh](/hack/install-tekton.sh)
+
+---
+
+Once the code have been modified, you can generate an instance of the Build operator running locally to validate your changes. Follow this step:
+
+For running the Build operator locally via the `local` target:
+
+```sh
+pushd $GOPATH/src/github.com/shipwright-io/build
+make local
```

Comment:
A `pushd` without a `popd` kind of feels lonely. I am not sure if other commands will follow or not, but if not, we could simplify it:
```suggestion
make -C $GOPATH/src/github.com/shipwright-io/build local
```

Reply:
done